docs/diagrams/SortActivityDiagram.puml

```diff
@@ -0,0 +1,17 @@
+@startuml
```

Comment:
This activity diagram has one extra weird arrow head. Look at the image generated

Reply:
Great spot!